federatedscope/core/trainers/trainer_FedRep.py

```diff
@@ -0,0 +1,94 @@
+import copy
+import torch
+import logging
+
+from federatedscope.core.auxiliaries.optimizer_builder import get_optimizer
+from federatedscope.core.trainers.torch_trainer import GeneralTorchTrainer
+
+from typing import Type
+
+logger = logging.getLogger(__name__)
+
+
+def wrap_FedRepTrainer(
+        base_trainer: Type[GeneralTorchTrainer]) -> Type[GeneralTorchTrainer]:
+
+    init_FedRep_ctx(base_trainer)
```

Comment:
Plz add some comments to remind the users that this method should be used in conjunction with `cfg.personalization.local_param` and `cfg.finetune.local_param`. Also the basic idea is should be introduced, e.g., which is to 
> train `cfg.personalization.local_param` parameter in the first `cfg.personalization.epoch_linear` rounds, and then train the other remaining parameters in the last `cfg.personalization.epoch_feature` rounds.